plugins/helm/parser/src/valueanalyzer.h

```diff
@@ -0,0 +1,71 @@
+#ifndef CC_PARSER_YAMLRELATIONCOLLECTOR_H
+#define CC_PARSER_YAMLRELATIONCOLLECTOR_H
+
+#include "yaml-cpp/yaml.h"
+
+#include "model/file.h"
+
+#include <model/microservice.h>
+#include <model/microservice-odb.hxx>
+#include <model/microserviceedge.h>
+#include <model/microserviceedge-odb.hxx>
+#include <model/helmtemplate.h>
+#include <model/helmtemplate-odb.hxx>
+
+#include <parser/parsercontext.h>
+
+namespace cc
+{
+namespace parser
+{
+
+class ValueAnalyzer
+{
+public:
+  ValueAnalyzer(
+    ParserContext& ctx_,
+    std::map<std::string, YAML::Node>& fileAstCache_,
+    uint64_t templateIdCounter);
+
+  void init();
+
+  ~ValueAnalyzer();
+
+private:
+  YAML::Node findValue(
+    std::string value_,
```

Comment:
Code smell: string parameter passed by value, instead of const reference. Is it required to make a copy here?